lib/internal/modules/esm/loader.js

```diff
@@ -306,7 +306,7 @@ class ESMLoader {
 
   /**
    * Collect custom/user-defined hook(s). After all hooks have been collected,
-   * calls global preload hook(s).
+   * the global preload hook(s) must be called.
```

Comment:
Cf [this thread](https://github.com/nodejs/node/pull/43772#discussion_r939971535):

> Because otherwise, since `addCustomLoaders` is now called multiple times (once for each loader), it would lead to `preload` being called multiple times, which in turn would execute the global preload hook multiple times, which would crash (you can try reverting this change; a test crashes).

Reply:
Sure, something like that'll do 🙂 (I'm suggesting adding a code comment)